### PR TITLE
DOCS-564: Move ML Model and add info for new version

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/services/ml/_index.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
 ---
-title: "Deploy a Model"
-linkTitle: "Deploy Model"
+title: "ML Models Service"
```
Comment (sguequierre) [inline docs/services/ml/_index.md:2]: Question: "ML Models" service vs "ML Model" service

Do you think you could make this consistent across the intro/titles here?

Comment (npentrel) [reply docs/services/ml/_index.md:2]: Ml models is what it's called in the product

Comment (npentrel) [reply docs/services/ml/_index.md:2]: Screenshot 2023-04-13 at 22 47 31

Comment (JessamyT) [reply docs/services/ml/_index.md:2]: Oh dear. Maybe we can ping frontend and change it there

**Review by sguequierre — APPROVED** [review]

File: docs/services/data/_index.md [hunk 2]
```diff
@@ -105,5 +105,5 @@ Once you have configured data capture and cloud sync, you can [view](../../manag
 
 ### Machine Learning
 
-You can use data synced to the cloud to [train machine learning models](../../manage/ml/train-model) and then [deploy these models to your robots](../../manage/ml/deploy-model) from the Viam App.
+You can use data synced to the cloud to [train machine learning models](../../manage/ml/train-model) and then [deploy these models to your robots](../../services/ml) from the Viam App.
```
Comment (JessamyT) [inline docs/services/data/_index.md:108]:
```suggestion
You can use data synced to the cloud to [train machine learning models](../../manage/ml/train-model) and then [deploy these models to your robots](../../services/ml) from the Viam app.
```

**Pull request merged**